data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+
+
+
+mission "Deep: Interrogation"
+	landing
+	source
+		attributes "deep"
+		attributes "spaceport"
+	to offer
+		has "Deep Archaeology 5: done"
+	
+	on offer
+		conversation
+			`When you land on <origin> and open your ship's main hatch, you find yourself face-to-face with two Deep Security officers, neither of which appear armed. "Hello, Captain <last>. We're going to need to ask you some questions. If you would please step off of your ship and follow us, that would be much appreciated."`
+			choice
+				`	(Follow their orders.)`
+					goto orders
+				`	"Why do you need to question me?"`
+				`	(Close the hatch and flee)`
+					goto flee
+			
+			`	"Because we wish to question you on the nature of your work for Albert Foster. Now please, step off your ship and come with us."`
+			choice
+				`	(Follow their orders.)`
+				`	(Close the hatch and flee.)`
+					goto flee
+			
+			label orders
+			`	You follow the officers to a building located on the eastern edge of the spaceport. You are led to a small, dimly lit room on the second floor that contains nothing but two chairs and a table. You are told to sit in one of the chairs and wait.`
+				goto questioning
+			
+			label flee
+			`	You attempt to close the hatch, but a strong tingling sensation radiates from your abdomen. As you drop to the floor, convulsing, you hold your eyes open long enough to glimpse a stun gun in one officer's hand.`
+			``
+			`	You wake up, slumped in a chair, in a dimly lit room. Across the table is another chair, empty. As you twist about, examining the rest of the room, the lingering soreness in your abdomen reminds you that even though you are alone for the moment, attempting to leave is likely unwise.`
+			
+			label questioning
+			choice
+				`	(Wait.)`
+			
+			`	Ten minutes pass in silence until a figure enters the room. In the dim light, you discern a tall, dark woman wearing what looks like a Deep Security uniform. Some of its badges, pins, and striping are unfamiliar to you.`
+			`	"I hope my men didn't have to struggle too much to get you here."`
+			choice
+				`	"Don't worry, it was just fine."`
+				`	"It wasn't the warmest welcome I've ever received."`
+			
+			`	The woman chuckles slightly. "No need to be sarcastic with me. Now, you know why you're here."`
+			choice
+				`	"I do?"`
+					goto clueless
+				`	"I didn't commit any crimes, so you have no right to detain me."`
+			
+			`	"Indeed no crimes were committed during your work for Albert Foster, but we still have the right to detain you to keep the Deep safe. Cause us no trouble during this little meeting, answer my question, and you will be on your way in a few minutes' time.`
```

Comment:
Recommend changing this section to:
```"Indeed, no crimes were committed during your work for Albert Foster, but we still have the right to detain you to keep the Deep safe. Cause us no trouble during this little meeting, answer my questions, and you will be on your way in a few minutes' time.```

Just adds a comma and makes "question" plural.